tests/field-block.dora

```diff
@@ -0,0 +1,21 @@
+//= cannon
+
+fun main() {
+  let a = A::empty();
+
+  assert(a.lx == 1);
+  assert(a.ly == 2);
+
+  assert(a.vx == 1);
+  assert(a.vy == 2);
+}
+
+class A(le: Int, va: Int) {
+  let lx: Int = { le + 1 };
+  let ly: Int = { self.lx + 1 };
```

Comment:
Not sure about this: For removing `nil`, I would like to remove access to `self` in the initializer expressions. This way we could easily ensure that there is no access to uninitialized fields (we can probably relax this restriction later). This test would introduce more usages of `self` and would make that harder.

Reply:
@dinfuehr done!